geode-core/src/distributedTest/java/org/apache/geode/management/internal/security/MultiUserAuthenticationDUnitTest.java

```diff
@@ -16,9 +16,14 @@
 package org.apache.geode.management.internal.security;
 
 import static org.apache.geode.cache.Region.SEPARATOR;
+import static org.apache.geode.test.awaitility.GeodeAwaitility.await;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
```

Comment:
I think you meant to use:
```
import static org.assertj.core.api.Assertions.assertThat;
```

Reply:
yeah, thanks!. I was wondering why some of the methods were not available.